LondonBritishLibrary/orient/BLorient5085.xml

```diff
@@ -0,0 +1,399 @@
+                                part of the Index of chapters are missing. Several leaves have been displaced.</note>
+                            <textLang xml:lang="gez"/>
+                            <msItem xml:id="ms_i1.1">
+                                <locus from="2ra"/>
+                                <title type="incomplete" ref="LIT1011Abusak#Chapter10"/>
+                                <note>The beginning is missing. The leaf of this chapter is misplaced.</note>
+                            </msItem>
+                            <msItem xml:id="ms_i1.2">
+                                <locus from="2ra"/>
+                                <title type="complete" ref="LIT1011Abusak#Chapter11"/>
+                                <note>The leaves with of chapter are misplaced.</note>
+                            </msItem>
+                            <msItem xml:id="ms_i1.3">
+                                <locus from="3va"/>
+                                <title type="complete" ref="LIT1011Abusak#Chapter12"/>
+                                <note>The leaves of this chapter are misplaced.</note>
+                            </msItem>
+                            <msItem xml:id="ms_i1.4">
+                                <locus from="6rb"/>
+                                <title type="complete" ref="LIT1011Abusak#Chapter13"/>
+                                <note>The leaves of this chapter are misplaced.</note>
+                            </msItem>
+                            <msItem xml:id="ms_i1.5">
+                                <locus from="8vb" to="11vb"/><locus from="20ra" to="20va"/>
+                                <title type="complete" ref="LIT1011Abusak#Chapter14"/>
+                                <note>The leaves of this chapter are partly misplaced.</note>
+                            </msItem>
+                            <msItem xml:id="ms_i1.6">
+                                <locus from="12ra" to="13va"/>
+                                <title type="incomplete">Index of the chapters</title>
+                                <note>The beginning (nos. 1-17) of the Index of chapters is missing. According to 
+                                    <bibl><ptr target="bm:Strelcyn1978BritishLibrary"/><citedRange unit="page">111</citedRange></bibl> the 
+                                    remaining parts of the index (nos. 18-59) occupy <locus from="2ra" to="13va"/>, but this is contradicted by Strelcyn's
+                                    information on the distribution of the chapters, which places ch. 10 on <locus from="2ra"/>, ch. 11 on 
+                                    <locus from="2ra"/>, ch. 12 on <locus from="3va"/>, ch. 13 on <locus from="6rb"/> and ch. 14 on 
+                                    <locus from="8vb" to="11vb"/>. For this reason, it can be assumed, that the index actually runs
```

Comment:
I don't know if we need to assume anything – maybe it's enough to simply state the facts, as you have done up to this sentence?

Reply:
Yes, it is better not to be vague. I think that 2va->12va is just a small change, but changing the loci of five subparts is just strange. That is why I favoured this solution. However, if this is too complicated and not convincing on first reading, it is better to delete this suppositional sentence.

Reply:
I looked at Strelcyn's description and I was confused. I could not figure out what he was intending to convey

Reply:
He gave an overview of the chapters following the numbers of chapters, but we need to follow the manuscript and number the msItems according to their appearance in the manuscript.